doc: regular updates (late Jan)

jorgeorpinel opened the pull request:
- Re-closes #855
- Closes #809
- Closes #686
- Closes #841
- Closes #506
- Other misc. stuff, as usual
- Closes #946

jorgeorpinel commented on public/static/docs/command-reference/import-url.md:
```diff
@@ -48,15 +47,20 @@ determine whether the local copy is out of date.
+| `ssh`    | SSH server                                            | `ssh://user@example.com:/path/to/data`     |
+| `hdfs`   | HDFS to file\*                                        | `hdfs://user@example.com/path/to/data.csv` |
+| `http`   | HTTP to file with _strong ETag_ (explanation below)\* | `https://example.com/path/to/data.csv`     |
+| `remote` | Remote path (see explanation below)                   | `remote://myremote/path/to/data`           |
```
Quick question @efiop. Only `import-url` supports remote:// URL notation? Asking because the `get-url` cmd ref doesn't have this line in the same table.

jorgeorpinel replied:
Seems like will need another update per iterative/dvc/pull/3214

jorgeorpinel replied:
I guess this is correct. I tried `dvc get-url remote://storage/get-started/data.xml` from inside my local clone of https://github.com/iterative/dataset-registry and got:
`ERROR: unexpected error - 'NoneType' object has no attribute 'repo'`

shcheklein replied:
this one looks like a bug we should create a ticket for in case you haven't done it yet

jorgeorpinel replied:
I think it makes sense that `get-url` doesn't support `remote://` URLs since it's meant to be used from anywhere, not just from inside DVC repos. Maybe just the error message could be better? I did open an issue about that: iterative/dvc/issues/3236 I will ask a Q there to see if the behavior is correct in the first place.

jorgeorpinel commented on public/static/docs/command-reference/get.md:
```diff
@@ -12,8 +12,8 @@ into the current working directory.
-  url         Location of DVC project or Git repository to download from
-  path        Path to a file or directory within the project or repository
+  url         URL of Git repository with DVC project to download from.
+  path        Path to a file or directory within a DVC repository.
```
Change contingent on iterative/dvc/issues/3196 (same in import.md)

jorgeorpinel replied:
Seems like will need another update per iterative/dvc/pull/3214.

jorgeorpinel replied:
OK done in d641dc84.

jorgeorpinel commented on public/static/docs/command-reference/checkout.md:
```diff
@@ -203,23 +204,25 @@ once to download missing data from the remote storage to the <abbr>cache</abbr>.
 
-We can then checkout the master branch again:
+> Note that the hooks only have an effect in repository versions (branches,
+> etc.) where DVC is active.
```
@shcheklein here's that note about the hooks (per https://github.com/iterative/dvc.org/issues/506#issuecomment-576053513).
BTW, I noticed `commit` and `push` commands don't have similar references to `dvc install`, should they? Maybe not a full blown example like in here but at least a note.

jorgeorpinel replied:
I decided to refine the existing notes and include it also in the other commands where it was missing (commit, repro, push).

jorgeorpinel commented on public/static/docs/tutorials/deep/sharing-data.md:
````diff
@@ -51,13 +51,14 @@ the master branch:
 $ git clone https://github.com/dmpetrov/new_tag_classifier.git
+$ cd new_tag_classifier
 $ dvc pull
 ```
````
Here's the fix to https://github.com/iterative/dvc.org/pull/938#issuecomment-576884513 (Dmitry restored the repo) `cd` was also missing.

shcheklein replied:
switched to example-get-started like was suggested?

jorgeorpinel replied:
Yes. Will be addressed in #946.

shcheklein replied:
👍

jorgeorpinel replied:
OK it was a small change so I did it here as well (54bcf5e5).

shcheklein commented on public/static/docs/command-reference/commit.md:
```diff
@@ -21,7 +21,13 @@ The `dvc commit` command is useful for several scenarios where a dataset is
+💡 For convenience, a Git hook is available to remind you to `dvc commit` when
+needed after a `git commit`. See `dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```
probably too much to include this note here

jorgeorpinel replied:
Removing...

shcheklein commented on public/static/docs/command-reference/checkout.md:
```diff
@@ -25,7 +25,12 @@ workspace data with the versions specified in the current DVC-files.
+[**Automating** example](#example-automating-dvc-checkout) below or
+`dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```
`Note that hook won't activate if the branch (commit, tag, etc) you checkout does not have DVC initialized (no .dvc directory)`?

jorgeorpinel replied:
Will use suggestion for https://github.com/iterative/dvc.org/pull/931#pullrequestreview-348280129, removing this one...

shcheklein commented on public/static/docs/command-reference/fetch.md:
````diff
@@ -185,6 +185,13 @@ $ tree .dvc
 ├── ...
 ```
 
+> The cache file with `.dir` extension is a special text file that contains the
````
probably not the purpose of fetch to explain exact cache structure, I keep it simpler

jorgeorpinel replied:
You're right. I'm just going to move the "Note that the `.dvc/cache` directory was created and populated." note from the command sample to a regular paragraph and leave the link to /doc/user-guide/dvc-files-and-directories#structure-of-cache-directory for more info... ⏳

jorgeorpinel replied:
Done as well. I also found this note in another few docs and removed/updated accordingly.

See 13e516ec

shcheklein replied:
so, my point is more about the `.dir` explanation. We don't need to have a note about it. It's fine to mention about cache structure.

jorgeorpinel replied:
Yep, I got it. I did remove the notes about .dir cache file and just left a note/link to cache structure 🙂

shcheklein commented on public/static/docs/command-reference/get-url.md:
```diff
@@ -36,14 +35,19 @@ single-purpose command that can be used out of the box after installing DVC.
+| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
+| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+
+> \* HDFS and HTTP **do not** support downloading entire directories, only
```
did we check re the HDFS one?

jorgeorpinel replied:
See https://github.com/iterative/dvc.org/pull/931#pullrequestreview-348279586 below. Updating...

jorgeorpinel replied:
> only local, s3, gs, ssh support dirs 

Dah ok that's all the remaining ones. No update needed! Resolving

shcheklein commented on public/static/docs/command-reference/get.md:
```diff
@@ -12,8 +12,8 @@ into the current working directory.
 positional arguments:
-  url         Location of DVC project or Git repository to download from
-  path        Path to a file or directory within the project or repository
+  url   Location of DVC project or Git repository to download from.
```
usually more spaces here?

jorgeorpinel replied:
This is to keep in a single line:

![image](https://user-images.githubusercontent.com/1477535/73569710-8c3abc80-4430-11ea-90d4-e508adbc4a71.png)

shcheklein replied:
but we already have a few commands with two lines? this format makes it somewhat harder to read I would say ... minor issue though

jorgeorpinel replied:
Yeah there's no exact rule about how many spaces we leave for these. Probably it's not very consistent. Since you feel there's not enough spaces I'll just add a few more haha and break the lines... ⌛️

jorgeorpinel replied:
OK done in 6f4de01f. I also updated a few more cmd refs for more consistent spacing (just up to index.md for now)

jorgeorpinel replied:
I'm extracting this into the next regular updates PR #966

shcheklein commented on public/static/docs/command-reference/import-url.md:
```diff
@@ -48,15 +47,20 @@ determine whether the local copy is out of date.
+| `http`   | HTTP to file with _strong ETag_ (explanation below)\* | `https://example.com/path/to/data.csv`     |
+| `remote` | Remote path (see explanation below)                   | `remote://myremote/path/to/data`           |
+
+> \* HDFS and HTTP **do not** support downloading entire directories, only
```
same re HDFS

jorgeorpinel replied:
Hmmm. Actually this is probably not true. I was just assuming because of the URL given above `hdfs://user@example.com/path/to/data.csv` which ends with a file extension, same as the HTTP example. I'll double check on this though, but I don't see a reason why we can't support HDFS dirs. Probably reverting, thanks!

jorgeorpinel replied:
I checked internally, @efiop confirmed indeed we do NOT support HDFS directories for these commands. In fact only local, s3, gs, ssh support dirs! Will update these refs.

jorgeorpinel replied:
Same here: No update needed! Resolving

shcheklein commented on public/static/docs/command-reference/import.md:
```diff
@@ -15,8 +15,8 @@ import.
 positional arguments:
-  url         Location of DVC project or Git repository to download from
-  path        Path to a file or directory within the project or repository
+  url   Location of DVC project or Git repository to download from.
```
same as with get, more spaces?

jorgeorpinel replied:
Same:

![image](https://user-images.githubusercontent.com/1477535/73569738-a1afe680-4430-11ea-9571-be240c6ef01a.png)

jorgeorpinel replied:
Will address along with https://github.com/iterative/dvc.org/pull/931#pullrequestreview-348279331 above.

shcheklein commented on public/static/docs/command-reference/install.md:
```diff
@@ -16,6 +16,9 @@ like Git to store code and configuration files. With `dvc install`, the two are
 more tightly integrated in order to cause certain convenient actions to happen
 automatically.
 
+> Note that the hooks only have an effect in repository versions (branches,
```
see the suggestion above, and probably this document is the only place we should keep this note. Also not sure if we need to make it `>` ... may be keep as a regular paragraph?

jorgeorpinel replied:
OK, using

```markdown
Note that this command requires the <abbr>DVC project</abbr> to be a Git
repository. But **the hooks won't activate** if the current branch (commit, tag,
etc.) doesn't have DVC initialized (no `.dvc/` directory present).
```

(not in block quote)

Pushed in 014e57f8.

shcheklein commented on public/static/docs/command-reference/push.md:
````diff
@@ -244,6 +250,11 @@ $ tree ../vault/recursive
 5 directories, 5 files
 ```
 
+> Cache files with `.dir` extension is a special text file that contains the
````
same as above

jorgeorpinel replied:
OK just left link to /doc/user-guide/dvc-files-and-directories#structure-of-cache-directory

See bf82ff22.

shcheklein commented on public/static/docs/command-reference/push.md:
```diff
@@ -29,6 +29,12 @@ The `dvc push` command allows one to upload data to remote storage. It doesn't
+💡 For convenience, a Git hook is available to automate running `dvc push` after
+`git push`. See `dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```
same as above

jorgeorpinel replied:
Removed

shcheklein commented on public/static/docs/command-reference/repro.md:
```diff
@@ -28,6 +28,12 @@ then be executed in the correct order, reproducing pipeline results.
+💡 For convenience, a Git hook is available to remind you to `dvc repro` when
+needed after a `git commit`. See `dvc install` for more details.
+
+> Note that the hooks only have an effect in repository versions (branches,
```
same here

jorgeorpinel replied:
Removed

shcheklein commented on public/static/docs/user-guide/dvcignore.md:
```diff
@@ -79,10 +80,16 @@ $ tree .dvc/cache
+<abbr>cached</abbr>. This means that `dvc add` ignored one of the files
+(`data1`).
+
+> The cache file with `.dir` extension is a special text file that contains the
```
same as above

jorgeorpinel replied:
Same. Just left note with link. See bf82ff22.

shcheklein commented on public/static/docs/user-guide/dvcignore.md:
```diff
@@ -44,9 +44,9 @@ it raises an error. Ignoring files inside such directory should be handled from
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example: Modification of ignored data
+## Example: Ignoring specific files
```
I think we should either keep the previous title (it was correct), or remove it completely (if it's a single example)

jorgeorpinel replied:
It's 1 out of 2 examples so we can either remove the subtitle or keep it. The previous one wasn't correct though, this example doesn't show any modification of previously ignored data. It's about how it looks to only ignore some files and other not.

jorgeorpinel replied:
But yeah its a pretty basic example so if you're not convinced with the new subtitle I can just remove it for this one, that works too.

jorgeorpinel replied:
OK I'm just removing the subtitle.

shcheklein commented on public/static/docs/user-guide/dvcignore.md:
```diff
@@ -44,9 +44,9 @@ it raises an error. Ignoring files inside such directory should be handled from
+## Example: Ignoring specific files
 
-Let's see what happens when we modify ignored file.
+Let's see what happens when we add a file to `.dvcignore`.
```
`add and then try to modify` ....

jorgeorpinel replied:
Oooooh I see what they meant with the previous subtitle now... That was confusing. Let me think about this ⏳

jorgeorpinel replied:
I'm not sure about this suggestion. If you keep reading below, there's this line:

> Now, let's modify file `data1` and see if it affects `dvc status`.

so that covers what you were trying to include here, I think. Especially now that the example's subtitle has been removed.

jorgeorpinel commented on public/static/docs/command-reference/import.md:
```diff
@@ -159,15 +159,10 @@ deps:
+If `rev` moves (e.g. a branch), you may use `dvc update` to bring the data up to
+date. If it's static however (e.g. a commit hash), it would be necessary to
+**re-import the data** instead, by using `dvc import` again with a different (or
+without) `--rev`. For example:
```
Here's the simplification I was talking about in https://github.com/iterative/dvc.org/pull/933#discussion_r373742627 @shcheklein

jorgeorpinel replied:
I assume this one is good 🙂

shcheklein replied:
`moves` is not clear enough. How about something like this:


If `rev` is a branch or tag name (any other <right git term>) it is usual that the most recent commit in the brach has updated, or tag now points to a different commit, etc. To update data and the DVC-file to point to the new commit in the branch or tag (<mention rev-lock?>) use ...

If `rev` is static `dvc update` does not have effect, you would need to re-import a new commit (or something like this).

jorgeorpinel replied:
OK. How about the newest version? (b0de4099)

```markdown
If `rev` is a branch, it will probably have newer commits periodically. To bring
the data up to date (and update `rev_lock` in the DVC-file), simply use
`dvc update`. If `rev` is more static (e.g. a tag or a commit hash) though, you
would need to **re-import** a different commit instead by using `dvc import`
again with a different (or without) `--rev`. For example:
```

shcheklein replied:
so, `dvc update` works with tags as well, it not that rate that people use them (like `latest` to designate the last to date model file).

otherwise looks good. I'm not sure if there is "static" term used in Git docs.

jorgeorpinel replied:
> dvc update works with tags as well... not that rare that people use them (like latest

I would argue it's pretty rare but OK, I came up with another version that is more complete, hopefully not overcomplicated again:

```markdown
If `rev` is a Git branch or tag, it may have new commits periodically. To bring
the data up to date (and update `rev_lock` in the DVC-file), simply use
`dvc update <stage>.dvc`. If `rev` is a specific commit (does not change),
`dvc update` will never have an effect on this import stage. You may
**re-import** a different commit instead, by using `dvc import` again with a
different (or without) `--rev`. For example:
```

Note that there's a feature request to add `--rev` to `dvc update` but it's not implemented yet: iterative/dvc/issues/2849 – we'll review this text again when that is implemented.

> not sure if there is "static" term used in Git docs

I don't know. I was just using general language 🙂 but that word is gone now anyway.

shcheklein replied:
`it may have new commits periodically` - tags may point to the new commit, not have them. Otherwise looks good :)

jorgeorpinel replied:
Changed to "it may point to a newer commit periodically" but it feels to technical or Git-specific again... ⏳

jorgeorpinel replied:
How about

```markdown
If `rev` is a Git branch or tag (where the commit it points to changes), the
data source may have updates at a later time. To bring it up to date if so (and
update `rev_lock` in the DVC-file), simply use `dvc update <stage>.dvc`. If
`rev` is a specific commit (does not change), `dvc update` will never have an
effect on the import stage. You may **re-import** a different commit instead, by
using `dvc import` again with a different (or without) `--rev`. For example:
```

(latest version in f409d81b)

shcheklein replied:
👍

jorgeorpinel commented on public/static/docs/glossary.js:
```diff
@@ -9,12 +9,9 @@ export default {
-form part of your expanded workspace, technically.
+code, ML models, etc. Note that
+[external outputs](/doc/user-guide/managing-external-data) also form part of
+your expanded workspace.
```
@shcheklein please also notice this change over in this PR as we recently discussed. Thanks

shcheklein replied:
I really don't like this note about external outputs (even if I suggested i, hah - I'm not sure where does it come from:) ). It would not make any sense to 100% readers. We can even test it with our team. Could recall what was the reason?

jorgeorpinel replied:
Sure, I can remove it. That note was already there though, this change removes the 2nd sentence (about becoming a DVC project). I'll test the external outputs sentence with the team meanwhile...

shcheklein replied:
test and ask if they are able to understand this :) Do you remember why was it added?

jorgeorpinel replied:
Just Ruslan answered, he said it makes sense to him. I think they will understand it but anyone who hasn't read about external data will not so you're right about that. But at the same time, this whole paragraph even including the note is pretty short now so I don't think we need to remove it. Anyone who reads this will get the idea from the previous sentence and if they don't understand this question they'll probably just ignore it (or click on the "external outputs" link). Up to you though, please lmk.

jorgeorpinel replied:
I don't remember why we added it. Probably had to do with reviewing or writing about the external outputs, cache, etc.

shcheklein replied:
Yep, that's expected from Ruslan :)

The problem with this that we mix so different things in the paragraph that anyone would think that this advanced one is of the same importance (while it's not). If we decide to keep, make it a paragraph, put `Advanced:`  or `in an advanced case ....`. So that it's clear for 99% of the readers it's super clear that they don't care about this.

jorgeorpinel replied:
Ok I've removed it and placed a similar but more correct note in the actual external output doc instead. See e6c244c4

> I also did small a revision of the other nearby glossary terms.

shcheklein commented on public/static/docs/command-reference/checkout.md:
```diff
@@ -25,7 +25,9 @@ workspace data with the versions specified in the current DVC-files.
 💡 For convenience, a Git hook is available to automate running `dvc checkout`
-after `git checkout`. Use `dvc install` to install it.
+after `git checkout`. See the
+[**Automating** example](#example-automating-dvc-checkout) below or
```
do we need to put into `**`? looks excessive a bit - emoji, link, bold

jorgeorpinel replied:
Haha no we don't. Idk why I did that. Bolding removed.

shcheklein commented on public/static/docs/user-guide/troubleshooting.md:
```diff
@@ -3,6 +3,8 @@
 In this section we provide help for some of the problems that DVC user might
 stumble upon.
 
+<!-- This file uses a special engine feature for the following headers, so that a custom anchor link is used. Just add {#custom-anchor} after each title: -->
```
no multi-line support for the comment?

jorgeorpinel replied:
Ah yeah sorry. I just had to add the first new line after `<!--` and then it rewraps it 😬. Fixed

shcheklein commented on public/static/docs/user-guide/contributing/docs.md:
```diff
@@ -154,3 +154,20 @@ pre-commit hook that is integrated when `yarn` installs the project dependencies
 > for example in
 > [this very file](https://raw.githubusercontent.com/iterative/dvc.org/master/public/static/docs/user-guide/contributing/docs.md).
+
+## General language guidelines
```
Would it make sense to formalize and start giving examples in terms how we choose terms - being human-friendly first, consistent (to a reason), etc?

jorgeorpinel replied:
OK good idea. I'm not totally sure what you have in mind but check out this paragraph I added in 6347a36f:

```markdown
We prefer human-friendly language than exact jargon, but we do seek preciseness,
even if using simple, general terms. Our
[command-references](/doc/command-reference) and some
[user-guides](/doc/user-guide) are the most technical documents we have, where
specialized terminology is used the most, but even there, often we use
expandable sections for complex implementation details.
```

Also not sure what you meant exactly by "start giving examples" though.

shcheklein replied:
`preciseness` sounds strange - correctness? not sure how to better put it


example: `prefer commit over revision whenever possible`
example: options in commands - start with the essence first, simple language, clarifications, edge cases, precision after ...

jorgeorpinel replied:
Updated to:

```markdown
We prefer human-friendly language than exact jargon, as long as it's correct,
even if using general terminology. Example: avoid Git jargon such as _revision_
or _reference_, preferring the more basic concept _commit_.

The [command-reference](/doc/command-reference) contains some of our most
technical documents where specialized language is used the most, but even there,
we use expandable sections for complex implementation details.

Start by writing the essence in simple terms, and complete it with
clarifications, edge cases, or other precisions in a separate iteration.
```

Better? Too long now?

shcheklein replied:
looks great! :)

shcheklein commented on public/static/docs/command-reference/repro.md:
```diff
@@ -28,6 +28,9 @@ then be executed in the correct order, reproducing pipeline results.
 > Pipeline stages are typically defined using the `dvc run` command, while
 > initial data dependencies can be registered by the `dvc add` command.
 
+💡 For convenience, a Git hook is available to remind you to `dvc repro` when
```
it's not necessarily `dvc repro` though. It might be `dvc commit` or `dvc checkout -f` or manually remove files ... it varies - we just run status, that's it

shcheklein replied:
so, not even sure it worth having this note here

jorgeorpinel replied:
Hmmm well, that's what the `dvc install` ref says it does: https://github.com/iterative/dvc.org/blob/master/public/static/docs/command-reference/install.md

But it sounds like I should try this and confirm that doc first.

jorgeorpinel replied:
> The installed Git hook automates reminding the user to run either dvc repro or dvc commit, as needed.

https://dvc.org/doc/command-reference/install

shcheklein replied:
that's should be easy enough to fix, it's two functions 5-10 lines of code each - https://github.com/iterative/dvc/blob/master/dvc/scm/git/__init__.py#L277 . The purpose of the initial doc was probably to clarify when it might be useful to see the status and fail (forgot to commit, repro, manually, may be even push (I would check this one), basically every situation status return non-zero exit code and what does it mean from a user perspective).

jorgeorpinel replied:
OK. I'm extracting this to a new issue #968

shcheklein commented on public/static/docs/command-reference/commit.md:
```diff
@@ -21,7 +21,10 @@ The `dvc commit` command is useful for several scenarios where a dataset is
-time tying stages or a pipeline.
+time tying stages or a pipeline. These scenarios are further detailed below.
+
+💡 For convenience, a Git hook is available to remind you to `dvc commit` when
```
I think it does not remind to this, it's just a regular status message.

jorgeorpinel replied:
Will follow up on all these notes (it's in several cmd refs) in https://github.com/iterative/dvc.org/pull/931#pullrequestreview-351859026 above.

jorgeorpinel commented on public/static/docs/install/linux.md:
````diff
@@ -42,7 +42,7 @@ In this case it installs `boto3` library as well, besides DVC.
 ```
 
-> Currently, it supports only Python versions 2.7, 3.6, and 3.7.
+> Currently, it supports only Python versions 3.6, and 3.7.
````
Does anyone know if this note is still current? Cc @efiop thanks

jorgeorpinel replied:
OK Ruslan confirmed that 3.5 3.6 and 3.7 are supported now (3.8 not yet)

jorgeorpinel replied:
Updated.

jorgeorpinel commented on public/static/docs/glossary.js:
```diff
@@ -25,14 +20,14 @@ Initialized by running \`dvc init\` in the **workspace** (typically in a Git
       match: ['DVC repository', 'DVC repositories'],
       desc: `
-**DVC project** initialized using \`dvc init\` in a Git repository. It will
+**DVC project** initialized (with \`dvc init\`) in a Git repository. It will
```
@shcheklein maybe we should merge the terms "DVC project" and "DVC repository" (into just project)? They're basically the same, but one of them specifies that it's in a Git repo.

shcheklein replied:
yep, good idea. And we can mention that it might or might not be Git controlled.

shcheklein replied:
or even mention there as well that some commands, options etc expect it to be Git as well as DVC and usually it's obvious from the context

jorgeorpinel replied:
OK! Terms merged in c55a2fe7 (updated all docs accordingly).

> we can mention that it might or might not be Git controlled

This is already mentioned in the definition of _DVC project_.

> some commands, options etc expect it to be Git as well as DVC and usually it's obvious from the context

Added this note into the definition. It's a little long now though:

![image](https://user-images.githubusercontent.com/1477535/73684419-c2bf4400-4689-11ea-8a8b-6157892dad03.png)

jorgeorpinel replied:
Should I remove the last sentence? "Some DVC commands..." I don't feel like it belongs in the short definition of the term. It will be mentioned in the appropriate cmd refs anyway.

jorgeorpinel replied:
Yeah. Removed in 7ef09685 (part of #966)

jorgeorpinel replied:
Discussion revived in https://github.com/iterative/dvc.org/pull/931#pullrequestreview-354892810.

jorgeorpinel commented on public/static/docs/install/macos.md:
```diff
@@ -63,9 +60,6 @@ In this case it installs `boto3` library as well, besides DVC.
-> Currently, it supports only Python versions 2.7, 3.6, and 3.7.
+> Currently, it supports Python 3.5-3.7
 
 <!-- Separate MD quote: -->
```
> can remove it now?
From https://github.com/iterative/dvc.org/commit/d1a6c129a65eff35907d0509044844e1dd1dad96#r37081978

Yes @shcheklein, thanks. Removing

shcheklein commented on public/static/docs/command-reference/get-url.md:
```diff
@@ -13,49 +12,55 @@ system.
-data files.
+<!-- Separate MD quote: -->
+
+> See `dvc get` to download data/model files or directories from other <abbr>DVC
```
three notes in a raw - smells bad )

jorgeorpinel replied:
Agree. And import-url has 4 in a row... Will think about this ⏳

jorgeorpinel replied:
Addressed in 1551b4b9 (part of #966).

shcheklein commented on public/static/docs/command-reference/get.md:
```diff
@@ -12,39 +12,37 @@ into the current working directory.
+target to be downloaded within the source repository at `url`. It can point to
+any file or directory in there, including <abbr>outputs</abbr> tracked by DVC,
+as well as files tracked by Git. Note that for DVC repos, the target should be
+found in one of the [DVC-files](/doc/user-guide/dvc-file-format) pf the project.
```
pf - of?

jorgeorpinel replied:
Addressed in #966

jorgeorpinel commented:
> few typos

Other than "pf"? (Addressed)

> unaddressed comments

Moving them to #966

> merging this anyway to prevent it from growing again

Thank you!

jorgeorpinel commented on public/static/docs/glossary.js:
```diff
@@ -8,33 +8,27 @@ export default {
+explained in each [command reference](/doc/command-reference) but usually
+obvious from the context.
       `
     },
```
These terms were merged per https://github.com/iterative/dvc.org/pull/931#pullrequestreview-351903751, which was the right move. But after a short discussion in https://github.com/iterative/dvc/pull/3277#discussion_r374425536 I think maybe we should make "DVC repository" (or just repo) the main or only term?

Initially I only thought of repos as the Git part of DVC projects, but later on the term DVC repo started popping up and we even had its glossary entry, now we've merged them both but left "project" as the main term. Comparing vs the core DVC repo though, the term "project" is not used at all so it would be easier to just manage the single term "repo" even when sometimes it will be necessary to disambiguate between Git vs DVC – a problem we currently have anyway.

Thoughts @shcheklein ? I can do this in my next regular updates or in its own PR (if it affects lots of files) if so.
